databricks/koalas/frame.py

```diff
@@ -661,8 +661,74 @@ def get(self, key, default=None):
         except (KeyError, ValueError, IndexError):
             return default
 
-    def sort_values(self, by):
-        return DataFrame(self._sdf.sort(by), self._metadata.copy())
+    def sort_values(self, by, ascending=True):
```

Comment:
ascending is Pandas is pretty sophisticated. You can also specify for each of the sort columns whether it is asc or desc. 

https://github.com/databricks/koalas/pull/53

Maybe you should take over this PR. It implements ascending and also na_position, inplace.

Reply:
Yup, will take that over.